httpserver/service_node.cpp

```diff
@@ -652,6 +628,23 @@ void ServiceNode::on_swarm_update(const block_update_t& bu) {
     initiate_peer_test();
 }
 
+void ServiceNode::relay_buffered_messages() {
+
+    // Should we wait for the response first?
+    constexpr std::chrono::milliseconds RELAY_INTERVAL = 500ms;
```

Comment:
2 differently scoped definitions of RELAY_INTERVAL for 200ms and 500ms
Should we just define in larger scope once or just make these the same? Or is the 500ms deliberate because it includes the actual relaying?

Reply:
Not deliberate, let's use 350ms for both as the average :)